src/main/kotlin/org/dashevo/dapiclient/model/GrpcExceptionInfo.kt

```diff
@@ -0,0 +1,34 @@
+package org.dashevo.dapiclient.model
+
+import io.grpc.StatusRuntimeException
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+
+
+class GrpcExceptionInfo (trailers: String) {
+    val errors = arrayListOf<Map<String, Any>>()
+
+    constructor(statusRuntimeException: StatusRuntimeException) : this(statusRuntimeException.trailers.toString())
+
+    init {
+        val cursor = trailers.findAnyOf(listOf("errors="))
+        val end = trailers.findLastAnyOf(listOf(")"))
+        val endpos = end?.first ?: trailers.length
+        if (cursor != null) {
+            val errorString = trailers.substring(cursor.first+ 7, endpos)
+            val json = "{errors: $errorString }"
+            val moshi = Moshi.Builder().build()
+            val map = Types.newParameterizedType(MutableMap::class.java, String::class.java, Any::class.java)
+            val adapter: JsonAdapter<Map<String, Any>> = moshi.adapter<Map<String, Any>>(map).lenient()
+            val errorMap = adapter.fromJson(json)
```

Comment:
Here you can use the fallowing code to remove trailing commas if needed:
```
    private fun removeTrailingCommas(json: String): String {
        return json.replace("\r\n", "\n")
            .replace(Regex(",[\\s]*]"), "]")
            .replace(Regex(",\\s*\\}"), "}")
    }
```

Reply:
great.  I will need that in android-dpp, which is where all the trailing commas are.  That will be a different PR!